test-clients/src/main/resource/DevTestnetAccount.txt

```diff
@@ -1 +1 @@
-rO0ABXNyABFqYXZhLnV0aWwuQ29sbFNlcleOq7Y6G6gRAwABSQADdGFneHAAAAADdwQAAAACdAANU1RBUlRfQUNDT1VOVHNxAH4AAAAAAAF3BAAAAAFzcgAxY29tLmhlZGVyYS5zZXJ2aWNlcy5sZWdhY3kuY29yZS5BY2NvdW50S2V5TGlzdE9iasKGpBBS1EebAgACTAAJYWNjb3VudElkdAAuTGNvbS9oZWRlcmFoYXNoZ3JhcGgvYXBpL3Byb3RvL2phdmEvQWNjb3VudElEO0wAC2tleVBhaXJMaXN0dAAQTGphdmEvdXRpbC9MaXN0O3hwc3IAN2NvbS5nb29nbGUucHJvdG9idWYuR2VuZXJhdGVkTWVzc2FnZUxpdGUkU2VyaWFsaXplZEZvcm0AAAAAAAAAAAIAA1sAB2FzQnl0ZXN0AAJbQkwADG1lc3NhZ2VDbGFzc3QAEUxqYXZhL2xhbmcvQ2xhc3M7TAAQbWVzc2FnZUNsYXNzTmFtZXQAEkxqYXZhL2xhbmcvU3RyaW5nO3hwdXIAAltCrPMX+AYIVOACAAB4cAAAAAIYAnZyACxjb20uaGVkZXJhaGFzaGdyYXBoLmFwaS5wcm90by5qYXZhLkFjY291bnRJRAAAAAAAAAAAAgAESgALYWNjb3VudE51bV9CABVtZW1vaXplZElzSW5pdGlhbGl6ZWRKAAlyZWFsbU51bV9KAAlzaGFyZE51bV94cgAmY29tLmdvb2dsZS5wcm90b2J1Zi5HZW5lcmF0ZWRNZXNzYWdlVjMAAAAAAAAAAQIAAUwADXVua25vd25GaWVsZHN0ACVMY29tL2dvb2dsZS9wcm90b2J1Zi9Vbmtub3duRmllbGRTZXQ7eHB0ACxjb20uaGVkZXJhaGFzaGdyYXBoLmFwaS5wcm90by5qYXZhLkFjY291bnRJRHNxAH4AAAAAAAF3BAAAAAFzcgAqY29tLmhlZGVyYS5zZXJ2aWNlcy5sZWdhY3kuY29yZS5LZXlQYWlyT2Jqfu50MIDYVscCAANMAAdwcml2S2V5dAAaTGphdmEvc2VjdXJpdHkvUHJpdmF0ZUtleTtMAApwcml2YXRlS2V5cQB+AAtMAAlwdWJsaWNLZXlxAH4AC3hwcHQAYDMwMmUwMjAxMDAzMDA1MDYwMzJiNjU3MDA0MjIwNDIwNjI2NjM3NjMzMzM0MzUzMTJkMzA2MTY2MzEyZDM0NjQzNjMwMmQzOTYyMzQ2NjJkMzYzMjYzNjYzNjM4NjMwMHQAQDZkZjk5ZGQ0OGJlMjQ5N2Y1ZWRkODczZWFiYTNlNDUwZWNiMDY3YTQ0NjJmNDk5OTliMDJhYzY1NTg4ZjM2NmF4eHg=
\ No newline at end of file
+rO0ABXNyABFqYXZhLnV0aWwuQ29sbFNlcleOq7Y6G6gRAwABSQADdGFneHAAAAADdwQAAAACdAANU1RBUlRfQUNDT1VOVHNxAH4AAAAAAAF3BAAAAAFzcgAxY29tLmhlZGVyYS5zZXJ2aWNlcy5sZWdhY3kuY29yZS5BY2NvdW50S2V5TGlzdE9iasKGpBBS1EebAgACTAAJYWNjb3VudElkdAAuTGNvbS9oZWRlcmFoYXNoZ3JhcGgvYXBpL3Byb3RvL2phdmEvQWNjb3VudElEO0wAC2tleVBhaXJMaXN0dAAQTGphdmEvdXRpbC9MaXN0O3hwc3IAN2NvbS5nb29nbGUucHJvdG9idWYuR2VuZXJhdGVkTWVzc2FnZUxpdGUkU2VyaWFsaXplZEZvcm0AAAAAAAAAAAIAA1sAB2FzQnl0ZXN0AAJbQkwADG1lc3NhZ2VDbGFzc3QAEUxqYXZhL2xhbmcvQ2xhc3M7TAAQbWVzc2FnZUNsYXNzTmFtZXQAEkxqYXZhL2xhbmcvU3RyaW5nO3hwdXIAAltCrPMX+AYIVOACAAB4cAAAAAIYAnZyACxjb20uaGVkZXJhaGFzaGdyYXBoLmFwaS5wcm90by5qYXZhLkFjY291bnRJRAAAAAAAAAAAAgAESgALYWNjb3VudE51bV9CABVtZW1vaXplZElzSW5pdGlhbGl6ZWRKAAlyZWFsbU51bV9KAAlzaGFyZE51bV94cgAmY29tLmdvb2dsZS5wcm90b2J1Zi5HZW5lcmF0ZWRNZXNzYWdlVjMAAAAAAAAAAQIAAUwADXVua25vd25GaWVsZHN0ACVMY29tL2dvb2dsZS9wcm90b2J1Zi9Vbmtub3duRmllbGRTZXQ7eHB0ACxjb20uaGVkZXJhaGFzaGdyYXBoLmFwaS5wcm90by5qYXZhLkFjY291bnRJRHNxAH4AAAAAAAF3BAAAAAFzcgAqY29tLmhlZGVyYS5zZXJ2aWNlcy5sZWdhY3kuY29yZS5LZXlQYWlyT2Jqfu50MIDYVscCAANMAAdwcml2S2V5dAAaTGphdmEvc2VjdXJpdHkvUHJpdmF0ZUtleTtMAApwcml2YXRlS2V5cQB+AAtMAAlwdWJsaWNLZXlxAH4AC3hwcHQAYDMwMmUwMjAxMDAzMDA1MDYwMzJiNjU3MDA0MjIwNDIwNjQ2NDYzMzczODM4NjYzNzJkMzczMTMwMzQyZDM0NjEzNzM5MmQzODYzMzk2NDJkNjI2MTMzNjMzMDYyMzMwMHQAQDNhZDJhMWY5MTFkZmUzNzQzODczNTc1ZTRlYjg1ZGM2ODhlYWIxM2QzYzlkMGVlM2I4N2FkYWVhMzhkNmJiNTN4eHg=
```

Comment:
I failed to see why we have to change this key every time we have a new saved state for migration.

Reply:
https://github.com/hashgraph/hedera-services/blob/77dd4e7834063eb486f294dfcc03e38e5d989bba/test-clients/src/main/java/com/hedera/services/bdd/suites/misc/RekeySavedStateTreasury.java#L66 

We always generate a new random key. 
I get your point on reusing the existing key. 
I don mind this step as it is just one more step of updating this file.

Reply:
Yeah, I was about think the same. It's probably easier for devops to give us a saved file with key already modified, as Quan mentioned several days ago. 

I guess this time it will be the way it is, but I suggest that we create an issue to get that procedure straightened out next release.

Reply:
In fact, we should get rid of this key. We should just use the genesis key for testing, including migration tests.